tsup.config.ts

```diff
@@ -1,13 +1,29 @@
 import { defineConfig } from 'tsup';
 
-export default defineConfig({
-  entry: ['src/**/*.ts'],
-  outDir: 'dist',
-  clean: true,
-  format: ['esm', 'cjs'],
-  target: ['es2020', 'node16'],
-  dts: false,
-  minify: false,
-  splitting: true,
-  bundle: false,
-});
+export default defineConfig([
+  // src esm
+  {
+    entry: ['src/index.ts'],
+    outDir: 'dist/esm',
+    clean: true,
+    format: 'esm',
+    target: ['es2020', 'node16'],
+    dts: false,
+    minify: false,
+    sourcemap: false,
```

Comment:
I don't know if it is relevant but I guess it would be useful to keep : 
- bundle: false
- sourcemap: true
- dts: true

It would ease the programmatic usage of the lib.

Reply:
`dts` is done in it's own step
https://github.com/salsita/node-pg-migrate/blob/0dacf09d34e1087a9a88e43b242b293850893849/package.json#L10

bundle and sourcemaps is currently only configured in this way for esm, not for cjs
So I plan to export everything needed from dist/index and depend on tree-shaking instead
But this will only be a breaking changes for users switching to esm anyway
I will drop cjs support in a later major

Reply:
Excellent !